packages/genesys-spark-components/src/components/beta/gux-avatar-group/gux-avatar-group-item/gux-avatar-group-item.scss

```diff
@@ -0,0 +1,122 @@
+@use '~genesys-spark/dist/scss/focus.scss';
+
+$gux-avatar-mask: "data:image/svg+xml,%3Csvg viewBox='0 0 26 26' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M22 13c0-2.6.7-5 1.8-7.2C21.5 2.3 17.5 0 13 0 5.8 0 0 5.8 0 13s5.8 13 13 13c4.5 0 8.5-2.3 10.8-5.8C22.7 18 22 15.6 22 13z' /%3E%3C/svg%3E";
+
+@mixin mask {
+  -webkit-mask-image: url($gux-avatar-mask);
+  mask-image: url($gux-avatar-mask);
+}
+
+::slotted(img) {
+  inline-size: 100%;
+  block-size: 100%;
+  object-fit: cover;
+}
+
+button {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  inline-size: var(--gse-ui-avatar-small-content-size);
+  block-size: var(--gse-ui-avatar-small-content-size);
+  padding: 0;
+  margin: 0;
+  margin-inline-end: var(--gse-ui-avatar-groupSet-gap);
+  overflow: hidden;
+  line-height: 0px;
+  color: var(--gse-ui-avatar-media-initialsForeground-inverse);
+  cursor: pointer;
+  background: none;
+  background-color: var(--gse-ui-avatar-media-initialsBackground-default);
+  border: none;
+  border-radius: 50%;
+
+  @include mask;
+
+  &:focus-visible,
+  &:hover {
+    position: relative;
+    z-index: var(--gse-semantic-zIndex-showFocus);
+    mask: none;
+
+    @include focus.gux-focus-ring;
```

Comment:
Should this styling just be for focus and not the hover styling ? In Figma it shows the tooltip but without the focus styling.